src/HTSPVFS.cpp

```diff
@@ -130,70 +88,40 @@ void CHTSPVFS::Close ( void )
   m_offset = 0;
   m_fileId = 0;
   m_path   = "";
-
-  Reset();
-}
-
-void CHTSPVFS::Reset()
-{
-  CLockObject lock(m_mutex);
-  m_buffer.reset();
-  m_bHasData = false;
-  m_bSeekDone = true;
-  m_currentReadLength = INITAL_READ_LENGTH;
-  m_seekCondition.Signal();
 }
 
 int CHTSPVFS::Read ( unsigned char *buf, unsigned int len )
```

Comment:
Did I really write a function using int's like this! urgh! really this should be something sensible like size_t

I was obviously having a bad day, or just putting up with how Kodi dev's had done things.